webpack.config.js

```diff
@@ -219,19 +113,11 @@ module.exports = {
         splitChunks: {
             cacheGroups: {
                 commons: {
-                    test: /[\\/]node_modules[\\/]/,
+                    test: /[\\/]node_modules[\\/].+\.(js|ts)$/, // split js only
                     name: 'vendors',
-                    chunks (chunk) {
-                        // exclude `style`
-                        return chunk.name !== 'style';
-                    }
                 }
             }
         },
-        minimizer: [
-            new CssMinimizerPlugin(),
```

Comment:
`sass-loader` minify generated CSS in production mode,
the CssMinimizerPlugin is not needed